packages/shared-ux/link/redirect_app/impl/src/redirect_app_links.component.tsx

```diff
@@ -44,7 +45,7 @@ export const RedirectAppLinks: FC<RedirectAppLinksComponentProps> = ({
 
   return (
     // eslint-disable-next-line jsx-a11y/click-events-have-key-events
-    <div ref={containerRef} onClick={handleClick}>
+    <div ref={containerRef} onClick={handleClick} className={className}>
```

Comment:
@elastic/shared-ux the div rendered here by `RedirectAppLinks` doesn't fill the vertical space available, so I could only make it work by implementing the `className` prop for this component and the ones it renders.

Reply:
Thanks - opened an issue for investigation on this in https://github.com/elastic/kibana/issues/140974